spec/acceptance/fragment_source_spec.rb

```diff
@@ -149,7 +149,6 @@
     end
     describe file("#{basedir}/fail_no_source") do
       #FIXME: Serverspec::Type::File doesn't support exists? for some reason. so... hack.
-      it { should_not be_file }
```

Comment:
why is this being removed? Shouldn't things that are not files still not be files?